designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/DeploymentManager.scala

```diff
@@ -80,7 +80,8 @@ trait DeploymentManager extends AutoCloseable {
   /**
     * We provide a special wrapper called WithDataFreshnessStatus to ensure that fetched data is restored
     * from the cache or not. If you use any kind of cache in your DM implementation please wrap result data
-    * with WithDataFreshnessStatus(data, cached = true)
+    * with WithDataFreshnessStatus(data, cached = true) => WithDataFreshnessStatus.cached(data) in opposite situation
```

Comment:
could be just "using `WithDataFreshnessStatus.cached(data)`, ... W don't need to expose this `WithDataFreshnessStatus(..., cached = true)` in this doc